tox.ini

```diff
@@ -21,6 +21,7 @@ pass_env =
     LANGCHAIN_PROJECT
     LLAMA_CLOUD_API_KEY
     NVIDIA_API_KEY
```

Comment:
this is for the notebooks, guess you can remove it

Reply:
I'll be adding an example notebook in my next PR. So I'll leave for now if that is okay.